rakelib/openhab.rake

```diff
@@ -9,7 +9,7 @@ require 'digest/md5'
 # rubocop: disable Metrics/BlockLength
 # Disabled due to part of buid / potentially refactor into classes
 namespace :openhab do
-  @openhab_version = '3.0.2'
+  @openhab_version = ENV['OPENHAB_VERSION'] || '3.0.2'
```

Comment:
perfect, that is how I was going to tackle it. 

Does that work for the RC versions?

Reply:
Good point, will have to add that to the regex, they should be in the same path as the milestone builds.

Reply:
The next issue I'm working on is to be able to have multiple versions installed side by side, to solve the caching in the github workers (and allowing for easier local testing). Having separate directories is no problem, the bigger issue is to ensure only one instance is running at a time.

Reply:
> The next issue I'm working on is to be able to have multiple versions installed side by side, to solve the caching in the github workers (and allowing for easier local testing). Having separate directories is no problem, the bigger issue is to ensure only one instance is running at a time.

I already took care of that in it is in main.

Reply:
Well, not the local part.. but it should work in github.

Reply:
Scratch that, didn't know you could have separate caches for the different versions, nice work!